### PR TITLE
Feature/1316 sub cache

### PR DESCRIPTION
**Pull request by kzangeli**
Last part of issue #1316 
No CNR entry added (already there)

@fgalan 
@crbrox

### CODE REVIEW

File: src/lib/apiTypesV2/SubscriptionExpression.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+struct SubscriptionExpression
+{
+  SubscriptionExpression(): isSet(false) {}
+
+  std::string               q;
+  std::string               geometry;
+  std::string               coords;
+  std::string               georel;
+  bool                      isSet;
+};
```
Comment (fgalan) [inline src/lib/apiTypesV2/SubscriptionExpression.h]: This has been moved from SubscriptionConetxtRequest. However, the following hasn't been moved:

```
Expression(): isSet(false) {}
```

Maybe it was a function not actually used?

Comment (kzangeli) [inline src/lib/apiTypesV2/SubscriptionExpression.h]: I don't fully understand.

    Expression(): isSet(false) {}

is now 

    SubscriptionExpression(): isSet(false) {}

Have I missed something?

Comment (fgalan) [inline src/lib/apiTypesV2/SubscriptionExpression.h]: Ok, now is *above* the fields instead *below* them... I misses that detail.

NTC

**Comment by fgalan** [comment]
LGTM

File: src/lib/apiTypesV2/SubscriptionExpression.h (hunk 1 above)
Comment (kzangeli) [inline src/lib/apiTypesV2/SubscriptionExpression.h]: Ah, yes, I moved it up ...

**Comment by crbrox** [comment]
LGTM

**Pull request merged**